### PR TITLE
ping: add Spanish translation

### PR DESCRIPTION
**Pull request by dagp-ucol**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by navarroaxel — COMMENTED** [review]
This page isn't in sync with https://github.com/tldr-pages/tldr/blob/main/pages/common/ping.md. Please update this PR.

**Review by blueskyson — APPROVED** [review]

File: pages.es/linux/ping.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ping
```
Comment (EmilySeville7cfg) [inline pages.es/linux/ping.md:1]:
```suggestion
# ping
```

Warning: `ping` was not found in `pages/linux/`.

File: pages.es/linux/ping.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> Comprueba la disponibilidad de un host.
+> Más información: <https://manpages.debian.org/bullseye/ping>.
+
+- Enviar paquetes ICMP a un host para comprobar que se encuentra disponible en la red.
+
+`ping {{host}}`
+
+- Enviar solo una cantidad específica de paquetes.
+
+`ping -c {{count}} {{host}}`
```
Comment (EmilySeville7cfg) [inline pages.es/linux/ping.md:12]:
```suggestion
`ping --count {{1..infinity}} {{host}}`
```

Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).
Optional fix: `--count` value can be converted to range (can't deduce the highest value for `--count`, `infinity` used as a fallback.

File: pages.es/linux/ping.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Enviar paquetes solo en un tiempo específico.
+
+`ping -w {{time}} {{host}}`
```
Comment (EmilySeville7cfg) [inline pages.es/linux/ping.md:16]:
```suggestion
`ping --timeout {{1..infinity}} {{host}}`
```

The same issue as above.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

**Comment by CleanMachine1** [comment]
If someone else would like to resubmit this PR, feel free, however due to the abandonment of it and the agreement isn't signed, I am closing this PR.

**Pull request closed without merging**